src/screens/CreatePost/elements/SheetAddTopic.js

```diff
@@ -176,7 +178,7 @@ const SheetAddTopic = ({refTopic, onAdd, topics, onClose, chatTopics}) => {
                         #{convertString(item.name, ' ', '')}
                       </Text>
                       {index !== topicSuggestion.length - 1 && (
-                        <View style={{height: 1, marginTop: 5, backgroundColor: '#C4C4C4'}} />
+                        <View style={{height: 1, marginTop: 5, backgroundColor: COLORS.gray1}} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color value has been updated to use a constant from the `COLORS` object. This change is consistent with the PR summary and improves maintainability by centralizing color definitions.
```diff
- <View style={{height: 1, marginTop: 5, backgroundColor: '#C4C4C4'}} />
+ <View style={{height: 1, marginTop: 5, backgroundColor: COLORS.gray1}} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->